supernode/services/artworkregister/task.go

```diff
@@ -796,6 +823,28 @@ func (task *Task) storeThumbnails(ctx context.Context) error {
 	return nil
 }
 
+func (task *Task) storeIDFiles(ctx context.Context) error {
+	store := func(ctx context.Context, ids []string) error {
+		for _, idFile := range ids {
+			if _, err := task.p2pClient.Store(ctx, []byte(idFile)); err != nil {
+				return err
+			}
+		}
+
+		return nil
+	}
+
+	if err := store(ctx, task.Ticket.AppTicketData.DDAndFingerprintsIDs); err != nil {
+		return fmt.Errorf("store ddAndFp files: %w", err)
+	}
+
+	if err := store(ctx, task.Ticket.AppTicketData.RQIDs); err != nil {
```

Comment:
any wrong here?
I don't think we store RQIDs into kademlia.
RQIDs here are hashs of RQIDs file - actualy is key to refer to p2p data of `SHA3-256(compress(Base64(rq_ids).Base64(signature).<cnt>))`.
I think you should store : `compress(Base64(rq_ids).Base64(signature).<cnt>))`

Reply:
correct, Good eyes! @nguyen202